src/libraries/System.Private.CoreLib/src/System/Text/Ascii.Utility.cs

```diff
@@ -1520,6 +1586,26 @@ private static Vector128<byte> ExtractAsciiVector(Vector128<ushort> vectorFirst,
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static Vector256<byte> ExtractAsciiVector(Vector256<ushort> vectorFirst, Vector256<ushort> vectorSecond)
```

Comment:
Is it worth having separate helpers for Narrow? I think it's pretty self-descriptive

Reply:
Yes, you are right, manually inlined the function with Vector API.